[Troubleshooting tip] - config.csp.frame-ancestors, split is not a function

### 1. Slack conversation link (if any)
Hi, I installed Zowe 1.13 with node.js version v8.16.0. After starting zowe server I saw following error message in appServer log.
```
failed to process config                                           
TypeError: config.csp.frame-ancestorsÝ0¨.split is not a function   
failed to process config                                           
TypeError: config.csp.frame-ancestorsÝ0¨.split is not a function   
failed to process config                                           
TypeError: config.csp.frame-ancestorsÝ0¨.split is not a function 
```  
I can logon to desktop, but when I open explorers I got following message.  This happened for all three explorers.
```
{"messages":[{"messageType":"ERROR","messageNumber":"ZWEAG708E","messageContent":"The request to the URL '/ui/v1/explorer-uss/' has failed after retrying on all known service instances. Caused by: java.net.ConnectException: EDC8128I Connection refused. (errno2=0x74940000) (Connection refused)","messageKey":"org.zowe.apiml.gateway.connectionRefused"}]}     
```             
Web developer tool showed a 500 error on GET to /ui/v1/explorer-jes/ . I installed 1.13 on node V12 didn’t hit this error. Could this issue related to Node.js version? Or is there anything else I missed? Thanks!

Response from @NakulManchanda 

We introduced this new variable
ZOWE_EXPLORER_FRAME_ANCESTORS in 1.13 if its undefined its treated as boolean and break string split function.
Please add it your instance.env, thanks!

Related issue opened for 1.15

https://github.com/zowe/explorer-ui-server/issues/44

### 2. Choose a title

Unable to log onto explorers when using Zowe 1.13 or 1.14 

### 3. Symptom

### 4. Solution

Should be fixed in 1.15 with issue 
https://github.com/zowe/explorer-ui-server/issues/44
This is to doc for troubleshooting

--- a/docs/troubleshoot/app-framework/app-troubleshoot.md
+++ b/docs/troubleshoot/app-framework/app-troubleshoot.md
@@ -265,3 +265,24 @@ ZWED0047W - Swagger file for service (org.zowe.terminal.tn3270:statediscovery) n
 
 **Solution:**   
 You can ignore these messages. 
+
+## Unable to log in to the explorers when using Zowe V1.13 or V1.14
+
+**Symptom:**
+
+You installed Zowe V1.13 or V1.14. When you start the Zowe server, you see the following error message in the `appServer` log. 
+
+```
+failed to process config                                           
+TypeError: config.csp.frame-ancestorsÝ0¨.split is not a function   
+```
+
+When you log in to the Zowe Desktop, you cannot open the JES, MVS, or USS Explorers. You receive the following error message: 
+
+```
+{"messages":[{"messageType":"ERROR","messageNumber":"ZWEAG708E","messageContent":"The request to the URL '/ui/v1/explorer-uss/' has failed after retrying on all known service instances. Caused by: java.net.ConnectException: EDC8128I Connection refused. (errno2=0x74940000) (Connection refused)","messageKey":"org.zowe.apiml.gateway.connectionRefused"}]} 
+```
+
+**Solution:**
+
+A new property `ZOWE_EXPLORER_FRAME_ANCESTORS` was introduced in V1.12. This property is required to be present in the `instance.env` file with some valid value. When undefined, it is treated as Boolean, which breaks the string split function. To resolve the issue, define the value for this property in the `instance.env` file. 

